autorest/codegen/serializers/builder_serializer.py

```diff
@@ -325,6 +325,14 @@ def param_description(  # pylint: disable=no-self-use
             for m in builder.parameters.method
             if m.method_location != ParameterMethodLocation.HIDDEN_KWARG
         ]:
+            # we don't want to show api_version in doc string even if tag of single-api package
+            # contains different api versions
+            if (
```

Comment:
I also think we can do this check earlier, i.e. on the parameter itself, instead of waiting all the way until builder serializer to check.

Also we don't want to check implementation. If the origin is the modelerfour:synthesized-api-version, we hide it

Reply:
Got it. After looking through the code, I find that we needn't check 'implementation`. https://github.com/Azure/autorest.python/pull/1239/files#diff-92eb2753e2f9a046e650ae5cb94a0bfad888868d08848e0bf80d815f1ce8fea1 shall be enough and it will not influence doc string of client